components/x-gift-article/src/ShareArticleDialog.jsx

```diff
@@ -6,7 +6,13 @@ import { AdvancedSharingBanner } from './AdvancedSharingBanner'
 
 export default (props) => {
 	return (
-		<div className="o-typography-wrapper share-article-dialog__wrapper" hidden={props.isLoading}>
+		<div
+			className="o-typography-wrapper share-article-dialog__wrapper"
+			hidden={props.isLoading}
+			data-trackable={`share-modal | ${
```

Comment:
**question (non-blocking):** is the use of ` | ` to delineate tracking values documented anywhere or is it just a convention? Every time I see it it confuses me until I remember that it isn't an 'OR' operator 😁

Reply:
It is used to separate the different values. I haven't seen it documented. If you take a look at an event in Amplitude you can see that when there are multiple values for one attribute they are separated with `|`. 

For example
<img width="400" alt="image" src="https://github.com/Financial-Times/x-dash/assets/22678655/856d0cf3-5a38-494d-a215-2fddb89c7750">